Add blog post for February standard community call (#153)
publiccodenet/blog: _posts/2021-02-11-notes-from-community-call-4-february-2021.md

---
title: "Notes from the Standard for Public Code community call - 4 February 2021"
date: 2021-02-11
author: Jan Ainali
type: blogpost
excerpt: How will certifications look like?
categories:
  - Community call
---

# Notes from the Standard for Public Code community call - 4 February 2021

## Attendees

* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
* [Eric Herman](https://publiccode.net/team/eric-herman.html)
* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)

## Updates from the Foundation

* Signalen is incubation for the Standard for Pubic Code

> * Signalen's and OpenZaak's assessments are both updated to the v.0.2.1 of the template

Reviewer: please hyperlink [v.0.2.1 of the template]

Author: Do we want to link to the empty template on our site, or each of the assessments?

Reviewer: Empty template!

* [Let's talk about public code #3 - Sergei Maertens and Tjerk Vaags, OpenZaak](https://podcast.publiccode.net/e/3-sergei-maertens-and-tjerk-vaags-openzaak/)
* We are in a lot of upcoming events:
    * [OFE Open Source Policy Summit](https://openforumeurope.org/event/policy-summit-2021/)
    * [Contributing (with) civil servants](https://fosdem.org/2021/schedule/event/community_devroom_contributing_with_civil_servants/)
    * FOSS Backstage:
        * [How the Standard for Public Code can help you collaborate over time and space](https://foss-backstage.de/session/how-standard-public-code-can-help-you-collaborate-over-time-and-space)
        * [The Governance Game - a conversation starter tool](https://foss-backstage.de/session/governance-game-conversation-starter-tool)

## Proposed for this call

* [How should we certify compliance?](https://github.com/publiccodenet/standard/issues/434)

## Notes

This community call was full of sharing visions of the future with a lot of ideas and bouncing them back and forth.
Here's a summary of how we currently believe certification will look like for codebases that comply with the Standard for Public Code.

> The Foundation for Public Code will store information about which versions of all codebases that are compliant with the Standard.

Reviewer: This sentence isn't quite right re English. Is this ok?
```suggestion
The Foundation for Public Code will store information organized by version for all codebases that are (or have ever been) compliant with the Standard.
```

Author: Hmm, I am not trying to state how they are organized, but that what we will store is references to which versions are compliant. They will probably be organized chronologically (that is, they will be added in the order they are created).

Reviewer: What about: "The Foundation for Public Code will store information for every version of all codebases that are (or have ever been) compliant with the Standard."
